### PR TITLE
Update chrony NTP test case

### PR DESCRIPTION
**Pull request by mloviska**
- [[qac] test fails in ntp_client](https://progress.opensuse.org/issues/106374) && [[sporadic]test fails in ntp_client - time could not be synced)](https://progress.opensuse.org/issues/95872)

- Verification runs:
* [sle-15-SP3-Server-DVD-Updates-x86_64-Build20220211-1-mau-extratests-security-fips@64bit](http://kepler.suse.cz/tests/13709) 200+ runs have passed
* [sle-15-SP3-JeOS-for-kvm-and-xen-Updates-x86_64-Build20220215-1-jeos-extratest@64bit-virtio-vga](http://kepler.suse.cz/tests/13710#live)
* [sle-15-SP4-JeOS-for-kvm-and-xen-x86_64-Build5.26-jeos-extratest@64bit-virtio-vga](http://kepler.suse.cz/tests/13711#)
* [opensuse-15.3-JeOS-for-kvm-and-xen-x86_64-Build9.371-jeos-extra@64bit_virtio-2G](http://kepler.suse.cz/tests/13712#live)
* [opensuse-15.4-JeOS-for-kvm-and-xen-x86_64-Build5.70-jeos-extra@64bit_virtio-2G](http://kepler.suse.cz/tests/13713)
* [opensuse-15.4-DVD-x86_64-Build140.1-extra_tests_textmode@64bit](http://kepler.suse.cz/tests/13714#live)
* [opensuse-Tumbleweed-JeOS-for-kvm-and-xen-x86_64-Build20220213-jeos-extra@64bit_virtio-2G](http://kepler.suse.cz/tests/13715)
* [tw-extra_tests_textmode](http://kepler.suse.cz/tests/13716)

### CODE REVIEW

File: tests/console/ntp_client.pm [hunk 1]
```diff
@@ -10,28 +10,28 @@ use strict;
...
-    if (is_sle() && !check_var('FLAVOR', 'JeOS-for-RaspberryPi')) {
+    if (is_sle() && !is_jeos) {
         systemctl 'enable chronyd';
         systemctl 'start chronyd';
-        # bsc#1179022 avoid '503 No such source' error while chrony does pick responding sources after start
-        script_run qq(timeout 85 bash -c 'until chronyc activity|grep "0 sources doing burst.*online"; do sleep 1; echo "waiting for ntp sources response"; done' -k);
+        my $inc = 0;
+        while (split(/\n/, script_output('chronyc sources') <= 3) && $inc < 10) {
+            sleep(++$inc);
+        }
```
Comment (pdostal) [inline tests/console/ntp_client.pm:29]: Can you comment this?

File: tests/console/ntp_client.pm [hunk 2]
```diff
@@ -10,28 +10,28 @@ use strict;
...
-use version_utils 'is_sle';
+use version_utils qw(is_sle is_jeos);
 
 sub run {
     my $self = shift;
     $self->select_serial_terminal;
 
     assert_script_run 'timedatectl';
+    # check if pool file exists
+    assert_script_run 'test -f /etc/chrony.d/pool.conf';
```
[the remark this answers is not in the document]
Comment (grisu48) [reply tests/console/ntp_client.pm:21]:
```suggestion
    assert_script_run('test -f /etc/chrony.d/pool.conf', fail_message => "chrony-pool package is missing");
```

Perhaps this is better then?

**Review by grisu48 — APPROVED** [review]

File: tests/console/ntp_client.pm (hunk 2 above)
[the remark this answers is not in the document]
Comment (mloviska) [reply tests/console/ntp_client.pm]: Done

**Comment by jlausuch** [comment]
Late LGTM :)

**Pull request merged**